### ISSUE TITLE
FoodPrints

### ISSUE THREAD

**Issue by zurda** [fac-15/project-ideas#5]
# FoodPrints

**tl;dr**: Eating in/ordering from zero waste and environment aware restaurants.

#### Name: 

Michal ([@zurda](https://github.com/zurda))

### The problem I'd like to solve

The average restaurant produces over 150,000 pounds of waste a year. Some restaurants are aware of this issue, and take a zero-waste mentality. This means they compost all food scraps, seek to use local ingredients that would have otherwise been trashed, and don't let anything wrapped in single-use packaging through the door.

I want to have an easy way to locate these restaurants so I can harm the environment as little as possible when eating out or ordering in. I also hope that increasing awareness might spread the word about this issue to restaurants and foodies. 

### Who are the stakeholders?
- **Zero waste restaurants** 
- **Environmentalists** 
- **Foodies**

#### Zero waste home

There are solutions to increase awareness to this issue and move to a zero waste life. For example, [Zero Waste Home](https://app.zerowastehome.com/) helps you fill your house with zero waste items. It has a [search option](https://app.zerowastehome.com/) for buying in bulk from zero waste stores.

#### Google reviews

You can google zero waste restaurants and find helpful info such as opening hours, and sometimes menus. Restaurants results might also contain reviews.

#### Online articles and posts

You can find many articles about zero waste restaurants online:
    
- [Vogue's Guide To The Best Zero-Waste Restaurants](https://www.vogue.co.uk/gallery/best-zero-waste-restaurants)
- [Could Zero-Waste Restaurants Be The Future Of Fine Dining?
](https://www.mindbodygreen.com/articles/nolla-zero-waste-restaurant)

These help share locations and veriety of specific restaurants, however it's usually hard to find an up to date restaurant list.
